recipe/recipe.yaml

```diff
@@ -0,0 +1,114 @@
+schema_version: 1
+
+context:
+  version: "11.3.0"
+  tests_to_skip: >
+    _not_a_real_test
+    ${{ "# compile smoke test failing to link python3x.lib" if 0 }}
```

Comment:
Please use the `interpreter: python` pattern as used for example on the numpy [feedstock](https://github.com/conda-forge/numpy-feedstock/blob/main/recipe/recipe.yaml)
```yaml
    script:
      interpreter: python
      content:
        - import numpy, sys
        - 'tests_to_skip = "_not_a_real_test"'
        # regular comment
        - [etc.]
```

Reply:
Oh nice, I wasn't a fan of these jinja hacks either. Let me give it a try.